_includes/manuals/nightly/5.5.1_backup.md

```diff
@@ -4,7 +4,7 @@ instance.
 
 #### Database
 
-##### PostgreSQL, MySQL, SQLite
+##### PostgreSQL, SQLite
```

Comment:
Maybe drop this header as well and decrease the `SQLite` disclaimer? Perhaps even reformat it to a generic Note since it's not intended for production.

Reply:
I'll remove the whole sqlite part, since it basically says "you can just back up the file but it is better to do a dump".